oblib/data_model.py

```diff
@@ -627,11 +585,19 @@ def _toJSON(self):
                 aspects["precision"] = str(self.precision)
 
         if isinstance( self.value, datetime.datetime):
-            value_str = self.value.strftime("%Y-%m-%dT%H:%M:%S")
+            # Format dates:
+            value_literal = self.value.strftime("%Y-%m-%dT%H:%M:%S")
```

Comment:
We should preserve localized times, when timezone is specified.  Suggest `.isoformat(sep='T')` instead of `.strftime("%....")`

Reply:
OK, done